mantidproject/mantid
# RROA now allows Qmin and Qmax to be specified separately

**Description of work.**
Originally if you only one of either MomentumTransferMax or MomentumTransferMin were specified it would be ignored. However it now takes one into account if the other is not present.

**Report to:** Jos and Max

**To test:**
- Run this script
```
LoadNexus(Filename='INTER00013460', OutputWorkspace='TOF_13460')
ReflectometryReductionOneAuto(InputWorkspace='TOF_13460', ThetaIn=0.69981140781600515, 
  MomentumTransferStep=0.03403761177182079, MomentumTransferMax=0.1, 
  OutputWorkspaceBinned='IvsQ_binned_13460', OutputWorkspace='IvsQ_13460', OutputWorkspaceWavelength='IvsLam_13460')
```
- Check the max X value in `IvsQ_binned_13460`. Note that it should no longer be `0.101672` and should be the `0.1` that was specified.
- Check the history of `IvsQ_binned_13460`. Note that `MomentumTransferMax` is `0.1` rather than the `0.101672` that it did previously.

Fixes #23119 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
